### DISCUSSION TITLE
@swc/core-android-arm-eabi not accessible from @swc/core

### DISCUSSION THREAD

**Discussion by krutoo** [swc-project/swc#3291] [Q&A] [answered]
Hi, im my CI job on self hosted gitlab script `npm ci` immediately fails with:

```
$ npm ci
npm ERR! @swc/core-android-arm-eabi not accessible from @swc/core
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-01-17T11_28_09_940Z-debug.log
```

How can i fix that error?

**Comment by kdy1** [comment, marked as answer]
Seems like your `package-lock.json` is out of sync?
https://stackoverflow.com/questions/55014020/running-npm-ci-returns-pkg-dir-not-accessible-from-find-cache-dir

You can run `npm i` once to sync it.

**Reply by krutoo** [reply]
`npm i` is not working, but with before removing `node_modules` with removing `package-lock.json` it works, thank you
